app/components/form/fields/DiskSizeField.tsx

```diff
@@ -32,6 +39,12 @@ export function DiskSizeField<
       min={minSize}
       max={MAX_DISK_SIZE_GiB}
       validate={(diskSizeGiB) => {
+        if (Number.isNaN(diskSizeGiB)) {
+          return 'Disk size is required'
+        }
+        if (imageSize && diskSizeGiB < imageSize) {
+          return `Must be as large as selected image (min. ${imageSize} GiB)`
+        }
```

Comment:
Another thing we could do here is just update the `min` input threshold. We'd probably want to include an info icon with some details to indicate _why_ the user can't go below the threshold though.

Reply:
Yeah, that’s why we went with two thresholds, the message saying it’s due to the image is pretty helpful. See also #1824